SSA/Projects/InstCombine/AliveStatements.lean

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
```suggestion

```

Stray newline?

Reply:
This file was added by accident. I dropped it now.